parachain/modules/relayer-fee-tracker/src/withdrawal.rs

```diff
@@ -0,0 +1,34 @@
+use codec::{Decode, Encode};
+use ismp::{host::StateMachine, messaging::Proof};
+use sp_core::H256;
+
+#[derive(Debug, Clone, Encode, Decode, scale_info::TypeInfo, PartialEq, Eq)]
+#[cfg_attr(feature = "std", derive(serde::Deserialize, serde::Serialize))]
+pub enum Key {
+    Request(H256),
+    Response((H256, H256)),
```

Comment:
lets make this a struct variant so that it's clear that they represent the `request_commitment` and the `response_commitment`